packages/components/src/components/button/button.custom.styles.ts

```diff
@@ -1,5 +1,100 @@
 import { css } from 'lit';
 
 export default css`
-  /* Write custom CSS here */
-`;
+
+  /*
+* SIZES
+*/
+  .button--medium {
+    font-size: var(--syn-button-font-size-large);
+  }
+
+  .button--large {
+    font-size: var(--syn-font-size-large);
+  }
+
+  .button--large syn-icon {
+    font-size: var(--syn-spacing-large);
+  }
+
+  .button.button--large.button--has-label.button--has-prefix .button__prefix,
+  .button.button--large.button--has-label.button--has-suffix .button__suffix {
+    font-size: var(--syn-spacing-large);
+  }
+
+  /*
+ * COLORING
+*/
+  .button--default.button--primary.button--disabled {
+    background-color: var(--syn-color-neutral-400);
+    border-color: var(--syn-color-neutral-400);
+    color: var(--syn-color-neutral-600);
+  }
+
+  .button--outline.button--primary.button--disabled {
+    background: none;
+    border-color: var(--syn-color-neutral-400);
+    color: var(--syn-color-neutral-400);
+  }
+
+  .button--text.button--primary.button--disabled {
+    color: var(--syn-color-neutral-400);
+  }
+
+  .button--default.button--primary:hover:not(.button--disabled) {
+    background-color: var(--syn-color-primary-900);
+    border-color: var(--syn-color-primary-900);
+    color: var(--syn-color-neutral-0);
+  }
+
+  .button--outline.button--primary:hover:not(.button--disabled),
+  .button--outline.button--primary.button--checked:not(.button--disabled) {
+    background: none;
+    border-color: var(--syn-color-primary-900);
+    color: var(--syn-color-primary-900);
+  }
+  
+  .button--text:hover:not(.button--disabled) {
+    color: var(--syn-color-primary-900);
+  }
+
+  .button:focus {
+    outline: var(--syn-focus-ring-color) solid 2px;
+    outline-offset: 2px;
+  }
+
```

Comment:
button:active seems to be missing here for the 'pressed' state.
And also active gets the keyboard-focus ring.

Reply:
Hi @Gwenevere ,
I'm sorry but my comment was completely misunderstanding. The 'pressed' state should NOT get the keyboard focus-ring and also the bg-color is darker: 
<img width="679" alt="button-pressed-state" src="https://github.com/SickDesignSystem/synergy/assets/142008971/2dc9b04e-80b9-4c4a-8b24-b2f6ad51c6e7">

Reply:
Ahhh, yeah, I misunderstood